test/integration/cluster_filter_integration_test.cc

```diff
@@ -44,6 +46,7 @@ class PoliteFilter : public Network::Filter, Logger::Loggable<Logger::Id::filter
   }
 
 private:
+  std::string greeting;
```

Comment:
nit: `const std::string greeting_;`

Reply:
Sorry you are missing the trailing underscore.

/wait

Reply:
Duh, thanks.